### ISSUE TITLE
How can i use "or" operator in regex expression in corpus query language ?

### ISSUE THREAD

**Issue by svjack** [liao961120/cqls#1]
I require following pattern match in corpus query language
python`re.findall("委托|代理", "把委托书或者代理文件")`
<img width="646" alt="截屏2021-07-10 下午4 45 57" src="https://user-images.githubusercontent.com/27874014/125157562-60337700-e19e-11eb-940f-d80fafff1db3.png">
but it seems not support, 
Do you have some ideas ?

**Comment by liao961120** [comment]
Yes, you are right, I did not implement the full set of CQL (e,g, `|` operation).
Given this pattern `[word="委托"]  | [word="代理"]` however, you can run two independent queries and concat the results together:
```python
cql1 = '[word="委托"]'
cql2 = '[word="代理"]'
```
